packages/astro/src/@types/astro.ts

```diff
@@ -822,6 +822,39 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		inlineStylesheets?: 'always' | 'auto' | 'never';
+
+		/**
+		 * @docs
+		 * @name build.split
+		 * @type {boolean}
+		 * @default {false}
+		 * @version 2.7.0
+		 * @description
+		 * Defines how the SSR code should be bundled.
```

Comment:
```suggestion
		 * Defines how the SSR code should be bundled when built.
```
Does this make sense to add? Is it too redundant? Your call!

Reply:
Makes sense!